enumerables.rb

```diff
@@ -0,0 +1,143 @@
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield (self[i])
+      i += 1
+    end
+  end
+
+  def my_each_with_index
```

Comment:
* [ ] There should be working `my_each_with_index`  identical and returns the same thing as ruby's [`each_with_index`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-each_with_index). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each_with_index.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
#### Let us compare the results return in each of the examples below and make the return the same result as the original method.
1. ```p [1,2,3,4,5].each_with_index``` and ```p [1,2,3,4,5].my_each_with_index``` must return same result.
2. ```p [1,2,3,4,5].each_with_index{|a| a}``` and ```p [1,2,3,4,5].my_each_with_index{|a| a}``` must return same result.
3. ```(0..5).each_with_index{|a,b| p b}``` and ```(0..5).my_each_with_index{|a,b| p b}``` must return same result.